### ISSUE TITLE
Support for InfluxDB v0.9

### ISSUE THREAD

**Issue by gnjack** [bernd/statsd-influxdb-backend#10]
InfluxDB 0.9 is coming soon, and [rc4](https://github.com/influxdb/influxdb/releases/tag/v0.9.0-rc4) has just been released which features an entirely new API and a tag system for describing metrics.

In addition to the new JSON format, the backend will need a way to define a retention policy and convert from a StatsD metric name into tag & field key value pairs.

Just hoping to start a discussion on how to approach this.

[InfluxDB v0.9.0-rc4 blog post](http://influxdb.com/blog/2015/02/24/influxdb-v0_9_0-rc4_released_now_with_tags_and_more_unicorns.html)

**Comment by imreACTmd** [comment]
I have a pull request for the v0.9 API support, it's currently pending at https://github.com/bernd/statsd-influxdb-backend/pull/14.

**Comment by bernd** [comment]
PR #14 has been merged. Can you please test if this works for you? Thanks!

**Comment by jaddison** [comment]
Just starting to read up on influx and statsd myself. Noticed this work lets statsd work with influxdb 0.9, but doesn't support tags. Is that right?

Although I'm not sure how it could work...

**Comment by imreACTmd** [comment]
yeah I don't see how statsd could support tags, but I'm open for ideas!

**Comment by linkslice** [comment]
one possible idea would to make set the tags in the config file. You already have to specify a db which means (at least in my environment) multiple statsd collectors anyway. If there was an item in the config file for tags to send along with the data when it flushes to influxdb.

**Comment by allankosciuszko** [comment]
version 0.9 implements new line protocol instead of JSON 
 https://github.com/influxdb/influxdb/pull/2696

**Comment by bernd** [comment]
Final JSON API changes have been merged with #17.

**Comment by tuukkamustonen** [comment]
Shouldn't this ticket be closed if the support has been implemented already?

**Comment by imreACTmd** [comment]
I haven't gotten to updating the code to work with the final version of the v0.9 api, hoping to get to it this week.

**Comment by tima** [comment]
@imreACTmd Are you referring to the new line protocol api in v0.9?

**Comment by tuukkamustonen** [comment]
How about parsing out tags from metric names, like InfluxDB graphite plugin can nowadays do? See https://github.com/influxdb/influxdb/blob/master/services/graphite/README.md

**Comment by n4nagappan** [comment]
+1 Would love to see the support for tags

**Comment by poll0rz** [comment]
+1 need tag support !!!

**Comment by berglh** [comment]
Some action is happening on [`this`]( https://github.com/Asana/statsd-influxdb-backend/commit/cc75147626498808d1b37ab0eb8afd410b9d515a?diff=unified) fork, but no documentation on how to utilise this yet - I'm guessing the `__t__` string is being used to specify the start of a tag, but I'm not sure this edit support mutl-value or multiple tags yet.
